scalafix/rules/src/main/scala/fix/v0_14_0/FixLogicalTypeSupplier.scala

```diff
@@ -0,0 +1,62 @@
+package fix.v0_14_0
+
+import scalafix.v1._
+import scala.meta._
+
+object FixLogicalTypeSupplier {
+  val ParquetConfigurationMatcher: SymbolMatcher = SymbolMatcher.normalized(
+    "com/spotify/scio/parquet/package/ParquetConfiguration#of"
+  )
+
+  val SetClassMatcher: SymbolMatcher = SymbolMatcher.normalized(
+    "org/apache/hadoop/conf/Configuration#setClass"
+  )
+
+  private val ParquetAvroPrefix = "com/spotify/scio/parquet/avro/syntax"
+  private val ParquetAvroReadMatcher = SymbolMatcher.normalized(s"$ParquetAvroPrefix/ScioContextOps#parquetAvroFile")
+  private val ParquetAvroWriteMatcher = SymbolMatcher.normalized(s"$ParquetAvroPrefix/SCollectionOps#saveAsParquetAvroFile")
+}
+
+class FixLogicalTypeSupplier extends SemanticRule("FixLogicalTypeSupplier") {
+  import FixLogicalTypeSupplier._
+
+  private def updateIOArgs(fnArgs: Seq[Term])(implicit doc: SemanticDocument): Seq[Term] = {
+    def removeTypeSupplier(confArgs: Seq[Term]): Option[Term] = {
+      val newConfiguration = confArgs.filterNot {
+        case q"$l -> classOf[LogicalTypeSupplier]" => true
```

Comment:
I couldn't figure out how to grab the symbol or signature of a `classOf` TypeRef, so left it as a string literal rather than a matched Symbol, but it feels a bit brittle...

Reply:
do this also match normal tuple ? (without the arrow syntax ?)